doc/source/install.rst

```diff
@@ -238,86 +196,85 @@ Query Analytics (QAN) requires:
       tar -xzf pmm-client.tar.gz
 
 3. Change into the extracted directory and run the install script.
-   Specify the IP address of the *PMM Server* host as the argument.
-   For example:
+   Specify the IP address of the *PMM Server* host
```

Comment:
Should we also show this?
`“sudo ./install <PMM server address[:port]> <client address>”`
It makes clear immediately what args are.